Issue building on Windows

I'm using Tauri v2, Linux and Mac build just fine, but windows gives me the error below. I will create a PR to fix this, all that needs done is on lib.rs, line 41, change `for window in &mut config.tauri.windows {` to `for window in &mut config.windows {`,  and line 91, change ` .join(&config.tauri.bundle.identifier)` to ` .join(&config.identifier)`



error[E0609]: no field `tauri` on type `&mut tauri::Config`
  --> C:\Users\runneradmin/.cargo\registry\src\index.crates.io-6f17d22bba15001f\tauri-plugin-theme-0.4.0\src\lib.rs:41:31
   |
41 |     for window in &mut config.tauri.windows {
   |                               ^^^^^ unknown field
   |
   = note: available fields are: `schema`, `product_name`, `version`, `identifier`, `app` ... and 3 others

error[E0609]: no field `tauri` on type `&tauri::Config`
  --> C:\Users\runneradmin/.cargo\registry\src\index.crates.io-6f17d22bba15001f\tauri-plugin-theme-0.4.0\src\lib.rs:91:27
   |
91 |             .join(&config.tauri.bundle.identifier)
   |                           ^^^^^ unknown field
   |
   = note: available fields are: `schema`, `product_name`, `version`, `identifier`, `app` ... and 3 others

For more information about this error, try `rustc --explain E0609`.
error: could not compile `tauri-plugin-theme` (lib) due to 2 previous errors

--- a/tauri-plugin-theme/src/lib.rs
+++ b/tauri-plugin-theme/src/lib.rs
@@ -38,7 +38,7 @@ pub fn init<R: Runtime>(_config: &mut Config) -> TauriPlugin<R> {
 #[cfg(target_os = "windows")]
 pub fn init<R: Runtime>(config: &mut Config) -> TauriPlugin<R> {
     let theme = saved_theme_value_from_config(&config);
-    for window in &mut config.tauri.windows {
+    for window in &mut config.app.windows {
         match theme {
             Theme::Auto => window.theme = None,
             Theme::Light => window.theme = Some(tauri::Theme::Light),
@@ -88,7 +88,7 @@ impl ToString for Theme {
 fn saved_theme_value_from_config(config: &Config) -> Theme {
     if let Some(dir) = dirs_next::config_dir() {
         let p = dir
-            .join(&config.tauri.bundle.identifier)
+            .join(&config.identifier)
             .join(CONFIG_FILENAME);
         return fs::read_to_string(p)
             .map(Theme::from)

